Update context section to talk about both SPacket and SFrame (#73)
sframe-wg/sframe: draft-ietf-sframe-enc.md

   scenarios.

# SFrame

> We propose an encryption mechanism that provides effective end-to-end encryption, is simple to implement, has no dependencies on RTP, and minimizes encryption bandwidth overhead. Because SFrame can encrypt a full frame, rather than individual packets, bandwidth overhead can reduced by adding encryption overhead only once per media frame, instead of once per packet.

Reviewer: We propose an encryption mechanism -> This document defines

> We propose an encryption mechanism that provides effective end-to-end encryption, is simple to implement, has no dependencies on RTP, and minimizes encryption bandwidth overhead. Because SFrame can encrypt a full frame, rather than individual packets, bandwidth overhead can reduced by adding encryption overhead only once per media frame, instead of once per packet.

Reviewer 2: ```suggestion
This document defines an encryption mechanism that provides effective end-to-end encryption, is simple to implement, has no dependencies on RTP, and minimizes encryption bandwidth overhead. Because SFrame can encrypt a full frame, rather than individual packets, bandwidth overhead can reduced by adding encryption overhead only once per media frame, instead of once per packet.
```

Reviewer 2: Suggesting the change proposed by @murillo128

> This document defines an encryption mechanism that provides effective end-to-end encryption, is simple to implement, has no dependencies on RTP, and minimizes encryption bandwidth overhead. Because SFrame can encrypt a full frame, rather than individual packets, bandwidth overhead can reduced by adding encryption overhead only once per media frame, instead of once per packet.

Reviewer 2: This is pre-existing, but we should probably wrap these lines by 80 characters like you are doing below.

## Application Context

SFrame is a general encryption framing that can be applied before or after

> packetization within a media stack.  That is, SFrame can be applied per-frame or

Reviewer 2: ```suggestion
packetization within a media stack. That is, SFrame can be applied per-frame or
```

> packetization within a media stack.  That is, SFrame can be applied per-frame or

Reviewer 2: I'd prefer we live packetization out of the this document. SFrame may be used in other places where there is no concept of packetization.

per-packet.  Applying SFrame per-frame offers higher efficiency, at the cost of a

> more complex integration with the media stack.  Applying SFrame per-packet makes

Reviewer 2: The more complex integration depends on the media stack implementation (see how easy it is in UAs to do frame level processing and how difficult it is in other native stacks).
